packages/guideline-blocks-settings/package.json

```diff
@@ -71,9 +71,20 @@
         "@frontify/sidebar-settings": "workspace:^",
         "@react-aria/focus": "^3.14.0",
         "@react-stately/overlays": "^3.6.1",
-        "@udecode/plate": "^21.5.0",
-        "slate": "^0.94.1",
-        "slate-react": "^0.98.3"
+        "@udecode/plate-core": "^30.1.2",
```

Comment:
Would prefer if everything of plate, slate, etc that is needed is exposed via `fondue`, that way it will be much cleaner in the future and also linking wouldn't be that messey

Reply:
I was thinking about that, but I see a couple issues.

1. There are some dependencies that are needed here but not needed in Fondue. E.g. `@udecode/zustood` or `@udecode/plate-normalizers`
2. If we want to add some changes to the plugins here, we would need to touch fondue if some components are not exported that was not needed before. Other approach is to export everything from fondue but then it would increase the bundle size.

What do you think?

Reply:
1. Maybe okay to just have them installed here, but would prefer to install them on fondue and expose the things that are needed of it.
2. Would only expose whats needed atm, and as it grows we will expose more via fondue. Not a big problem that we need to do fondue release first imo

Maybe worth to discuss it with Noah and/or Sam, but I really wanna have a clean state after the bump 😅